index.js

```diff
@@ -4,10 +4,19 @@
 module.exports = {
   name: 'ember-nvd3-shim',
   included: function included(app) {
+
     if (process.env.EMBER_CLI_FASTBOOT !== 'true') {
       app.import(app.bowerDirectory + '/d3/d3.min.js');
       app.import(app.bowerDirectory + '/nvd3/build/nv.d3.js');
-      app.import(app.bowerDirectory + '/nvd3/build/nv.d3.css');
+
+      var options1 = app.options['ember-cli-nvd3'];
+      var options2 = app.options['ember-nvd3-shim'];
```

Comment:
@offirgolan - basically stole your code here, and am checking for the presence of `includeCss` option on two objects now: `ember-nvd3-shim` or `ember-cli-nvd3`. If a non-false option is present on __either__, the css will be included.